utils/classify.py

```diff
@@ -3,17 +3,21 @@
 from utils.constants import c_org_names, f_companies, f_org_names
 
 
-def classify_wrapper(individuals_df, organizations_df):
+def classify_wrapper(
+    individuals_df: pd.DataFrame, organizations_df: pd.DataFrame
+):
     """Wrapper for classificaiton in linkage pipeline
 
     Initialize the classify column in both dataframes and
```

Comment:
elaborate what it means to be classified as f, c, neutral (i.e. neutral is the default and when we classify as f or c we are X confident that the entity is correctly classified)